manifests/package.pp

```diff
@@ -9,12 +9,16 @@
     path    => '/usr/bin:/usr/sbin:/bin:/usr/local/bin',
   }
 
-  $caddy_url    = 'https://caddyserver.com/download/linux'
-  $caddy_dl_url = "${caddy_url}/${caddy::params::arch}?plugins=${caddy::caddy_features}&license=${caddy::caddy_license}&telemetry=${caddy::caddy_telemetry}"
-  $caddy_dl_dir = "${caddy::params::caddy_tmp_dir}/caddy_linux_${$caddy::params::arch}_custom.tar.gz"
+  $caddy_url        = 'https://caddyserver.com/download/linux'
+  $caddy_dl_url     = "${caddy_url}/${caddy::params::arch}?plugins=${caddy::caddy_features}&license=${caddy::caddy_license}&telemetry=${caddy::caddy_telemetry}"
+  $caddy_dl_dir     = "${caddy::params::caddy_tmp_dir}/caddy_linux_${$caddy::params::arch}_custom.tar.gz"
+  $caddy_dl_command = $caddy_license ? {
+    'personal'   => "curl -o ${caddy_dl_dir} \"${caddy_dl_url}\"",
```

Comment:
Can't we use singlequotes within the string instead of escaped double quotes?

Reply:
changed, will test and commit if ok